libs/cardreader/build.gradle

```
@@ -0,0 +1,43 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.0"
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

```suggestion
            minifyEnabled true
```

I think we might want to enable code obfuscation for release build type. Additionally, I'd consider removing the `release` variant of the module, I'm not sure if there's a reason to have it.